### PR TITLE
Update quotes

### PR DESCRIPTION
**Pull request by Ngoakor12**
Pull-Request for `paretOS`

## Description
- Refactored structure of quotes to easily incorporate mutiple quotes from the same author therefore reducing duplication.
- Added getRandomQuote function in same directory to work with the new quotes structure.
- Added 5 new quotes.

## Reviewers
- @mikhael28

### CODE REVIEW

File: src/libs/quotes.ts [hunk 1]
```diff
@@ -2,37 +2,52 @@
...
   {
-    quote:
-      "Those who are not courageous enough to take risks will accomplish nothing.",
     author: "Muhammad Ali",
+    quotes: [
+      "Those who are not courageous enough to take risks will accomplish nothing.",
+    ],
   },
 ];
+
```
Comment (jayeclark) [inline src/libs/quotes.ts:58]: This structure (random author and then random author's quote) is great - a contributor who's a big fan of a particular author can add in a bunch of quotes for them, but it won't cause the unintended side where 90% of the quotes that are served up are from that author. :)

**Review by jayeclark — CHANGES_REQUESTED** [review]
Great PR, thank you so much for the contribution! The new flow of getting and using quotes is much cleaner.

I had only one very minor change request, related to yarn.lock.

File: yarn.lock [hunk 2]
```diff
@@ -3200,6 +3200,11 @@
   resolved "https://registry.yarnpkg.com/@types/mime/-/mime-1.3.2.tgz#93e25bf9ee75fe0fd80b594bc4feb0e862111b5a"
   integrity sha512-YATxVxgRqNH6nHEIsvg6k2Boc1JHI9ZbH5iWFFv/MTkchz3b1ieGDa5T0a9RznNdI0KhVbdbWSN+KWWrQZRxTw==
 
+"@types/minimist@^1.2.0":
```
[the remark this answers is not in the document]
Comment (Ngoakor12) [reply yarn.lock:3203]: Thanks for having a look, appreciate the feedback. I'll merge in new changes now now.

**Comment by mikhael28** [comment]
Thank you @Ngoakor12 !

**Comment by mikhael28** [comment]
And you of course @jayeclark for reviewing :D

**Pull request merged**